### ISSUE TITLE
Terrible perfomance in DOOM on FF

### ISSUE THREAD

**Issue by caiiiycuk** [DustinBrett/x#11]
Hi. I am author of js-dos project, I am very glad too see that your amazing site is using js-dos. However in my environment DOOM works very slow in Firefox (chrome is totally fine). Can you confirm? 

This problem only exists on your site, because on dos.zone performances are good both in Chrome in FF.

**Comment by DustinBrett** [comment]
Hi, I haven't actually done much testing in Firefox. Thanks for js-dos, it's amazing! It's possible the VantaJs animated wallpaper is slowing everything down. Thank you for bringing this issue to my attention either way, I've actually just started a total rebuild of this project which I am streaming on YouTube. When I get to adding DOS again I will do more cross browser and performance testing.

**Comment by DustinBrett** [comment]
Hey @caiiiycuk,

Just wanted to update you that I've recently installed js-dos (https://github.com/DustinBrett/x/commit/748c2c7f9932e1aaf6f4496b0afc846b4bada166) into v2 of my project and live streamed (https://youtu.be/Hu1RYVoCbgs?t=7055) the process. I tested it with Firefox after my stream and at least for me it's running Doom smoothly. Hopefully the speed is improved in this attempt. Thanks again for the great project!
